packages/storybook8/stories/Composites/ChatComposite/Docs.mdx

```diff
@@ -151,6 +151,7 @@ details. File sharing in Teams interop scenarios is available with the CallWithC
 <SingleLineBetaBanner />
 
 The following example show how to enable rich text editor for message editing by providing the `richTextEditor` property.
+The insert inline images feature is only supported in the Teams Interop chats. Please try it out in the Join Teams Meeting page.
```

Comment:
Should link to this page? Or does it not exist yet?

Reply:
> Should link to this page? Or does it not exist yet?

@palatter  Yes, Storybook 8 doesn't have a CallWithChat page yet.